README.md

```diff
@@ -2,3 +2,5 @@
 
```

Comment:
1. Update readme with screenshot of the application modules.
2. Provide the sample templates built on codepen